synfig-core/src/synfig/loadcanvas.cpp

```diff
@@ -3359,7 +3360,46 @@ CanvasParser::parse_canvas(xmlpp::Element *element,Canvas::Handle parent,bool in
 					}
 					
 				}
-				canvas->set_meta_data(child->get_attribute("name")->get_value(),content);
+
+				// Guide lines storage changed:
+				// Before:
+				//   guide_x -> list of X positions of vertical guide lines (separated by spaces)
+				//   guide_y -> list of Y positions of horizontal guide lines (separated by spaces)
+				// Now:
+				//   guide -> list of X*Y*angle coordinates of guide lines (data from same guide is separated by star *) (items separated by spaces)
+				if (meta_name == "guide_x") {
+					std::string new_content;
+					std::istringstream iss(content);
+					std::string item;
+					while (std::getline(iss, item, ' ')) {
+						if (!item.empty()) {
+							new_content += item + "*0*1.57079633 ";
+						}
+					}
+
+					content = new_content + " " +canvas->get_meta_data("guide");
+					if (content.back() == ' ')
+						content.pop_back();
+					meta_name = "guide";
+				}
+
+				if (meta_name == "guide_y") {
+					std::string new_content;
+					std::istringstream iss(content);
+					std::string item;
+					while (std::getline(iss, item, ' ')) {
+						if (!item.empty()) {
+							new_content += "0*" + item + "*0 ";
+						}
+					}
+
+					content = new_content + " " +canvas->get_meta_data("guide");
```

Comment:
> content = new_content + " " +canvas->get_meta_data("guide");

Do we really need to add a space here? 
If we made the conversion, then we already have them, if not, then it seems like we don’t need them.

Reply:
You're right! Done